components/ui/SetupPicker.tsx

```diff
@@ -29,6 +29,7 @@ export const SetupPicker: Picker = (props: PickerProps<unknown>) => {
 
   return (
     <Column
+      testID={props.testID}
```

Comment:
This is the wrapper of rendering items like list of languages.
Need to add testId to each list item and that has to be unique for each item.

Reply:
refactored this